pkg/core/util.go

```diff
@@ -59,10 +59,12 @@ func createGenesisBlock(cfg config.ProtocolConfiguration) (*block.Block, error)
 	scriptOut := hash.Hash160(rawScript)
 
 	minerTx := transaction.NewMinerTXWithNonce(2083236893)
+	minerTx.ValidUntilBlock = 1
```

Comment:
I don't think it's a valid value for these transactions. NEO 3 genesis block transactions don't set it in any way and given our regular transaction flow they're kinda verified at height -1, because they appear in block 0, so it's OK for them to have `ValidUntilBlock == 0`.